### ISSUE TITLE
Hardcoded French?

### ISSUE THREAD

**Issue by girzel** [emencia/emencia-django-newsletter#20]
The two templates:

newsletter_link_site.html
newsletter_link_unsubscribe.html

Seem to contain hardcoded (no i18n translation) French strings for viewing the newsletter onsite or unsubscribing. Surely this isn't the way it's meant to be? Am I supposed to override these templates with my own, or wouldn't it make sense to use translated strings here...

**Comment by Fantomas42** [comment]
Hi taojian,

thank you for your report, effectively, these strings are hardcoded in French... 

At first, I thought I have made a mistake, but I remembered that these templates are included without language information, because they are used without the request, directly when the email's content is generated, so translations are not possible.

Perhaps my conclusions are wrong, so I need to retest the behavior of these templates, with the _trans_ tag.

For you, the quickest solution is to override these templates, in your language, because I have not the time this week to make the necessary debugging.

Have a nice day.
